deployment/core-infrastructure/kustomize-templates/key-manager/01-rbac.yaml

```diff
@@ -67,7 +67,7 @@ metadata:
   name: key-manager-inference-services
 rules:
 - apiGroups: ["serving.kserve.io"]
-  resources: ["inferenceservices"]
+  resources: ["inferenceservices", "llminferenceservices"]
```

Comment:
This will also be needed in the `key-manager/deploy`. I hope to unify this sooner than later.

Reply:
I added in 3769dc1.